pysense/lib/LIS2HH12.py

```diff
@@ -95,34 +92,59 @@ def pitch(self):
         rad = -math.atan2(y, (math.sqrt(x*x + z*z)))
         return (180 / math.pi) * rad
 
+    def set_register(self, register, value, offset, mask):
```

Comment:
The idea is that the offset is used to shift the value as well as the mask (to set bit 4 to 1, one uses value = 1, mask = 1, offset = 4, not value = 0x10, mask = 0x10). This allows passing directly things like ODR or scale, rather than having to shift them in the calling function. Recomputing the offset from an already-offset mask would be quite inefficient.